_dashboards/geojson-regionmaps.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Region map visualizations
+nav_order: 7
+---
+
+# Region map visualizations
+
+OpenSearch Dashboards provides basic map tiles with a standard vector map that you can use to create your region map visualizations. You can configure the base map tiles using the WMS map server.
+
+You can't configure a server to support user-defined vector map layers. However, you can configure your own GeoJSON file and upload it for this purpose. 
+{: .note }
```

Comment:
is this a placeholder? seems like something is missing here 😅

Reply:
The {: .note} and {: .tip} are style markers in Markdown. It displays correctly on the website.

Reply:
Ah got it! Thanks : )